src/Library/Controller.php

```diff
@@ -95,4 +95,14 @@ protected function response(HttpHandler $response, $qualified, $mode = HttpHandl
 
         return $response;
     }
+
+    /**
+     * Return if the environment is Sandbox
+     *
+     * @return int getSandbox
+     */
+    public static function getSandbox()
+    {
+        return $this->configuration->getSandbox();
+    }
```

Comment:
Hey mate, this code is not being used, this was a block from #82 . Could you remove this block, please?

Reply:
Sure, will do.